.github/workflows/test-all-custom-flags.yml

```diff
@@ -144,3 +148,32 @@ jobs:
             ./test-summary
             ./tests-out/videos
             ./tests-out/screenshots
+
+      # Only post back to the PR if there was a failure since test-all has the
+      # typical config case covered for success feedback
```

Comment:
Reasoning here being, we don't need to be overly noisy if everything looks good, but we still want to alert if these cases fail.